.github/workflows/rust.yml

```diff
@@ -36,6 +36,16 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
+      - name: Cargo Cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
```

Comment:
Does caching a home directory create any race conditions when multiple runners are going at the same time?

Reply:
I suppose there is some kind of lock that exists in another way it doesn't make sense.